pypyr-nodejs/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM pypyr/pypyr:1.1.1
+
+ENV NODE_VERSION 10.13.0
+ARG NPM_VERSION=6.4.1
+ARG YARN_VERSION=1.10.1
+
+USER root
```

Comment:
I'm wondering about switching back to `pypyruser` at the very end of the Dockerfile. I've a suspicion this'll get permission errs when trying to run a pipeline that uses node/npm, though?